docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3971_cloud_region/3971_3.md

```diff
@@ -0,0 +1,41 @@
+- Perform [3971_1](3971_1.md)
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page >> **CLOUD REGIONS** tab
+3. Select `region1` on the left panel >> In the **Mount rules:** section select option `Same region` for *Object storages* and *File storages*. Click **Save** button.
+4. Select `region2` on the left panel >> In the **Mount rules:** section select option `None` for *Object storages* and *File storages*. Click **Save** button.
+
+| Steps | Actions | Expected results |
+| :---: |--- | --- |
+| 1 | Open the **Tools** page | |
+| 2 | Select test tool | |
+| 3 | Run tool with *Default settings* | |
+| 4 | At the Runs page, click the just-launched run | |
+| 5 | Wait until the **SSH** hyperlink appears | |
+| 6 | Click the **SSH** hyperlink | |
+| 7 | In the opened tab, enter and perform the <br> `pipe run -di <tool>:latest -cmd "sleep infinity" -pt on-demand -r <region1_id> -- CP_CAP_LIMIT_MOUNTS "<storage1_id>,<storage2_id>,<nfs_storage1_id>" -y` <br> command, where `<tool>` is name of test tool | The output contains `Pipeline run scheduled with RunId: <runID>` |
+| 9 | Store `runID` from output as `run1_id` | | 
+| 10 | Perform the <br> `pipe run -di <tool>:latest -cmd "sleep infinity" -pt on-demand -r <region2_id> -- CP_CAP_LIMIT_MOUNTS "<storage1_id>,<storage2_id>,<nfs_storage1_id>" -y` <br> command, where `<tool>` is name of test tool | The output contains `Pipeline run scheduled with RunId: <runID>` |
+| 11 | Store `runID` from output as `run2_id` | |
+| 12 | Open the **Runs** page | |
+| 13 | Click the run with *ID* `run1_id` | |
+| 14 | Expand *Parameters* section | *Parameters* section contains the text `CP_CAP_LIMIT_MOUNTS: <storage1_name>,<storage2_name>,<nfs_storage1_name>` |
+| 15 | Wait until *MountDataStorages* task is completed | |
+| 16 | Click on *MountDataStorages* task at the left panel | *MountDataStorages* task log is displaying that contains the following text: <br> `Run is launched with mount limits (<storage1_id>,<storage2_id>,<nfs_storage1_id>). Only 2 storages will be mounted.` <br> `Found 2 available storage(s). Checking mount options.` <br> 2 records: `-->{storage_name} mounted to /cloud-data/{storage_name}`, <br> where `{storage_name}` is `storage1`, `nfs_storage1` name |
+| 17 | Open the **Runs** page | |
+| 18 | Click the run with *ID* `run2_id` | |
+| 14 | Expand *Parameters* section | *Parameters* section contains the text `CP_CAP_LIMIT_MOUNTS: <storage1_name>,<storage2_name>,<nfs_storage1_name>` |
+| 15 | Wait until *MountDataStorages* task is completed | |
+| 16 | Click on *MountDataStorages* task at the left panel | *MountDataStorages* task log is displaying that contains the following text: <br> `Run is launched with mount limits (<storage1_id>,<storage2_id>,<nfs_storage1_id>). Only 0 storages will be mounted.` <br> `Found 2 available storage(s). Checking mount options.` <br> `No remote storages are available or CP_CAP_LIMIT_MOUNTS configured to none` |
```

Comment:
I could be wrong but I suppose that record `Found 2 available storage(s). Checking mount options.` will not be in the task log in such case.